### PR TITLE
Cypress/testing setup

### PR DESCRIPTION
**Pull request by Afoucheaux**
## Purpose
Cypress testing of all major feature. 
## Approach
Cypress doc's.
Turing class and notes.
#### Open Questions and Pre-Merge TODOs
- [x] Check functionality of site.
- [x] Note any Sad paths that we may need to add.
- [x] Compare to spec to verify happy path testing is complete. Note any missing.
## Learning
Basic understanding of Cypress testing.

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (elizhahn) [inline cypress.json:1]: que es?

Comment (elizhahn) [reply cypress.json:1]: oh I think this might be one of the many files cypress loads

**Review by elizhahn — APPROVED** [review]

**Comment by elizhahn** [comment]
In general I think we need to flush out the testing more. We will need to add testing for forward and backward clicks, and the loading error for single movie as well as a failed load error.

**Comment by elizhahn** [comment]
Additionally the background image needs to be tested and the mouseover

File: cypress.json (hunk 1 above)
Comment (Afoucheaux) [reply cypress.json:1]: I think you are correct about the cypress files. We can reach out and see what we actually need to keep.

**Pull request merged**